packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -1,28 +1,23 @@
 .auto-crud {
   display: flex;
   overflow: hidden;
+  border: solid 1px var(--lumo-contrast-20pct);
+}
+
+.auto-crud vaadin-split-layout::part(splitter) {
+  border-top: solid 1px var(--lumo-contrast-20pct);
+  border-left: solid 1px var(--lumo-contrast-20pct);
 }
 
 .auto-crud-main {
   flex: 1 1 100%;
+  min-width: 200px;
```

Comment:
Using a min-width for the main panel containing the grid so that you can't collapse it completely.